python/lsst/meas/pz/estimate_pz_task.py

```diff
@@ -64,28 +67,22 @@ class EstimatePZConnections(PipelineTaskConnections, dimensions=("instrument", "
     )
 
     objectTable = cT.Input(
-        doc="Object table in parquet format, per patch",
-        name="objectTable",
+        doc="Object table in parquet format",
+        name="object",
```

Comment:
I don't know if this change is ok given that we're between the two naming conventions. I _think_ that for now the preference is to update https://github.com/lsst/drp_pipe/blob/main/pipelines/_ingredients/base-v2.yaml so that tasks can be run using any of the supported instruments (not all instrument pipelines use the new naming scheme). There was some discussion this week as to how long we are going to support both naming schemes and my understanding was that it will at least continue for the time being.

Reply:
Yes... this isn't in any DRP pipelines yet, though, and won't be until the package moves to `lsst` at least, so I'm mostly indifferent about it. At worst we'll just have to change it back to `objectTable` on a subsequent ticket.